lua/damagelogs/server/autoslay.lua

```diff
@@ -12,7 +12,7 @@ if not sql.TableExists("damagelog_autoslay") then
 		ply varchar(32) NOT NULL,
 		admins tinytext NOT NULL,
 		slays SMALLINT UNSIGNED NOT NULL,
-		reason tinytext NOT NULL,
+		reason varchar(255) NOT NULL,
```

Comment:
This change _should_ be fine.
It won't affect anyone who already has this table in their server database.

I'm not sure if there are any significant differences between these two types in the version of SQLite that GMod uses.
Possibly around allowed default values.